tests/modules/matrix_atttention/matrix_attention_tests.py

```diff
@@ -0,0 +1,35 @@
+import torch
+from numpy.testing import assert_almost_equal
+import numpy as np
+from allennlp.common.testing.test_case import AllenNlpTestCase
+from allennlp.modules.matrix_attention import CosineMatrixAttention, DotProductMatrixAttention
```

Comment:
This would be better as two different files, with class names that match what we've done in the rest of the repo.